docs/i18n.rst

```diff
@@ -62,16 +62,11 @@ We don't expect translators to deal with all these files
 directly. Translation happens on our `Weblate`_ server, which is
 configured to use a fork of the `main SecureDrop repository`_.
 
-At the start of the :ref:`release process <release_management>`, the
-localization manager collects string changes on the ``develop`` branch
-in the `main SecureDrop repository`_ and merges them to the ``i18n``
-branch of the `securedrop-i18n repository`_. The changes will then
-appear in Weblate, and translation can begin. At the end of the
-translation period, the localization manager collects the changes to
-the PO files on ``securedrop-i18n/i18n`` in a pull request for
-``securedrop/develop``. Once that pull request is merged, the
-translations are backported to the release branch in the `main
-SecureDrop repository`_.
+As string changes are merged into the ``develop`` branch
+in the `main SecureDrop repository`_, the changes will automatically
+appear in Weblate, and translation can begin. Translations take
+place continuously, at any time. Translations for a release are
+reviewed and finalized during the release process.
```

Comment:
Some things to clarify and/or expand:

```suggestion
appear in Weblate, and translation can begin. Translation and review can take
place continuously, at any time. Translations for supported languages [link] are
reviewed and finalized during the release process.
```

As in <https://github.com/freedomofpress/securedrop-client/wiki/Internationalization>, I think our goal should be that translations get code-merged more rather than less often.  This will mean (a) [some translations get code-merged before they've been translation-reviewed](https://github.com/freedomofpress/securedrop-client/pull/1548#discussion_r1156375077) and (b) some translations will be merged for unsupported languages, before language support is evaluated as part of the release process.